tools/src/main/java/org/apache/kafka/trogdor/common/Topology.java

```diff
@@ -33,7 +33,7 @@
         public static Set<String> agentNodeNames(Topology topology) {
             Set<String> set = new HashSet<>();
             for (Map.Entry<String, Node> entry : topology.nodes().entrySet()) {
-                if (Node.Util.getTrogdorAgentPort(entry.getValue()) > 0) {
+                if (entry.getValue().getConfig(Platform.Config.TROGDOR_AGENT_PORT) != null) {
```

Comment:
I think the test failure in the PR builds are related to this change. `MiniTrogdorCluster` used in the tests sets agent port to zero on the coordinator, so this check doesn't work?

Reply:
Good point.  Fixed